pkg/systemd/init.js

```diff
@@ -1269,16 +1269,18 @@ $(function() {
         var unit = "[Unit]\nDescription=";
         var service = "\n[Service]\nExecStart=";
         var timer = "\n[Timer]\n";
-        var service_file = unit + timer_unit.Description + service + timer_unit.Command + "\n[Install]\nWantedBy=default.target";
+        var install = "[Install]\nWantedBy=default.target\n";
```

Comment:
Yes, that is what made me look at this in the first place.  `default.target` stopped working with the systemd version currently in Fedora 26.  I am not sure whether that is a regression in systemd, or whether `WantedBy=default.target` was never supposed to work.

What about using `WantedBy=basic.target`?